.github/workflows/ci.yml

```diff
@@ -160,7 +159,7 @@ jobs:
           "$SDKMANAGER" 'build-tools;28.0.3' | tr '\r' '\n' | uniq
           "$SDKMANAGER" 'platforms;android-26' | tr '\r' '\n' | uniq
           "$SDKMANAGER" 'extras;android;m2repository' | tr '\r' '\n' | uniq
-          "$SDKMANAGER" ndk-bundle | tr '\r' '\n' | uniq
+          "$SDKMANAGER" 'ndk;21.3.6528147' | tr '\r' '\n' | uniq
```

Comment:
Not really this CL, but do we need to pin the NDK version?  Also I guess we should consider a newer cmake?

Reply:
The newer Android gradle plugin is opinionated about NDK in 3.6+. You can either figure out what version it wants and put it here or explicitly pull one in. The version that it wants by default is something around 21.0.xxxx which is pretty old.

Reply:
Got it, thanks